tests/unit/lms/services/mailchimp_test.py

```diff
@@ -14,21 +14,15 @@
 
 
 class TestSendTemplate:
-    def test_it(self, mailchimp_transactional, caplog):
+    def test_it(
+        self, svc, mailchimp_transactional, mailchimp_client, caplog, sender, recipient
+    ):
         caplog.set_level(logging.INFO)
-        svc = MailchimpService(sentinel.api_key)
```

Comment:
This constructor call was duplicated in lots of tests so I moved it into an `svc` fixture